src/app/api/query/base.ts

```diff
@@ -52,14 +50,16 @@ export function useWebsocketAwareQuery<
   >
 ) {
   const queryClient = useQueryClient();
-  const connectedCount = useSelector(statusSelectors.connectedCount);
   const { subscribe } = useWebSocket();
-
   const queryModelKey = Array.isArray(queryKey) ? queryKey[0] : "";
 
-  useEffect(() => {
-    queryClient.invalidateQueries();
-  }, [connectedCount, queryClient, queryKey]);
+  // TODO: Investigate why the code below causes the zones endpoint to be polled 50 times on page load
+  // https://warthogs.atlassian.net/browse/MAASENG-3558
+
+  // const connectedCount = useSelector(statusSelectors.connectedCount);
+  // useEffect(() => {
+  //   queryClient.invalidateQueries();
+  // }, [connectedCount, queryClient, queryKey]);
```

Comment:
It looks like this might be happening because a new `queryKey` array reference is passed in on each render even though the array is always `["zones"]` and useEffect reruns when a reference is changed instead of checking the array's contents.

You could probably fix this bug by either storing the query key in a way that means the reference doesn't change (e.g. as a variable outside the hooks in zones.ts or inside a ref) or by checking whether the queryKey content changes and updating a ref inside this hook.